src/app-instance.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * app-timestamp.js
```

Comment:
Should this file be named app-timestamp.js then?

Reply:
I had originally called it app-timestamp and decided to call it app-instance. The fact that it uses a timestamp is really just an implementation detail. I'll fix the comment.